rollup.config.ts

```diff
@@ -33,15 +33,20 @@ const config: RollupOptions[] = [
         skip: ['bsv'],
         browser: true,
         preferBuiltins: true,
-        moduleDirectories: ['node_modules'],
       }),
       commonjs(),
       json(),
-      typescript({ tsconfig: "./tsconfig.json", sourceMap: false }),
+      typescript({ 
+        tsconfig: "./tsconfig.json", 
+        sourceMap: false,
```

Comment:
Can we have source map true? maybe this will make it easier to debug the code from client perspective?

Reply:
Yes we can consider it - but not now please (at least not in this PR 😄)